### PR TITLE
cleanup: change the .md formatter to wrap at 80 characters

### PR DESCRIPTION
**Pull request by alevenberg**
I didn't notice anything that becomes unreadable with this change. Let me know if I'm missing something.

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/12252)

### CODE REVIEW

File: .mdformat.toml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-wrap = "keep"       # possible values: {"keep", "no", INTEGER}
+wrap = 80           # wrap all .md files to 80 characters - possible values: {"keep", "no", INTEGER}
```
Comment (coryan) [inline .mdformat.toml:1]: checkers did not like the extra space at the end of this line.

File: doc/packaging.md
[the remark this answers is not in the document]
Comment (alevenberg) [reply doc/packaging.md:1]: The markdown tool won't lint over the .dockerfile

Leaving as is for now

File: .mdformat.toml (hunk 1 above)
Comment (alevenberg) [reply .mdformat.toml:1]: Fixed

Comment (alevenberg) [reply .mdformat.toml]: My branch has the commit (https://github.com/googleapis/google-cloud-cpp/compare/main...alevenberg:google-cloud-cpp:md), but it's not in the PR....

Comment (alevenberg) [reply .mdformat.toml]: Never mind, just a delay

**Review by dbolduc — APPROVED** [review]
Nice, I won't have to think about this ever again.

File: CHANGELOG.md [hunk 2]
```diff
@@ -170,26 +170,34 @@ the APIs in these libraries are stable, and are ready for production use.
 
 ### [Batch](/google/cloud/batch/README.md)
 
-- docs: an example to extract Cloud Batch logs ([#11935](https://github.com/googleapis/google-cloud-cpp/pull/11935))
-- docs(batch): introduce basic snippets ([#11915](https://github.com/googleapis/google-cloud-cpp/pull/11915))
+- docs: an example to extract Cloud Batch logs
+  ([#11935](https://github.com/googleapis/google-cloud-cpp/pull/11935))
+- docs(batch): introduce basic snippets
+  ([#11915](https://github.com/googleapis/google-cloud-cpp/pull/11915))
```
[the remark this answers is not in the document]
Comment (alevenberg) [reply CHANGELOG.md:176]: [see](https://screenshot.googleplex.com/7qM5yTGUZ6B8MZR)

**Pull request merged**